src/index.ts

```diff
@@ -0,0 +1,18 @@
+export class Challenge
+{
+    public buzz<Promise>(a: number)
+    {
+        return new Promise((resolve, reject) =>
+        {
+            if(a % 3 === 0)
+                if(a % 5 === 0)
+                    resolve('fizzbuzz');
+                else 
+                    resolve('fizz');
+            else if(a % 5 === 0)
```

Comment:
🙈
We usually use the bracket for block that are composed by multiple lines
```suggestion
            if(a % 3 === 0)
            {
                if(a % 5 === 0)
                    resolve('fizzbuzz');
                else 
                    resolve('fizz');
            }
            else if(a % 5 === 0)
```